x-pack/plugins/security_solution/public/common/components/event_details/summary_view.tsx

```diff
@@ -135,6 +141,45 @@ const getSummary = ({
           linkValue: linkValue ?? undefined,
         };
 
+        if (item.id === 'signal.threshold_result.terms') {
+          try {
+            const terms = getOr(null, 'originalValue', field);
+            const parsedValue = terms.map((term: string) => JSON.parse(term));
+            const thresholdTerms = (parsedValue ?? []).map(
+              (entry: { field: string; value: string }) => {
+                return {
+                  title: `${entry.field} [threshold]`,
+                  description: {
+                    ...description,
+                    value: entry.value,
+                  },
+                };
+              }
+            );
+            return [...acc, ...thresholdTerms];
+          } catch (err) {
+            return acc;
+          }
+        }
+
+        if (item.id === 'signal.threshold_result.cardinality') {
+          const parsedValue = JSON.parse(value);
```

Comment:
This can throw exceptions, I believe. Value is literally a signal field value here?

Reply:
@banderror It is an object with `field` and `value` fields.

Reply:
Thank you